Source/SwiftLintFrameworkTests/Yaml+SwiftLintTests.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  Yaml+SwiftLintTests.swift
+//  SwiftLint
+//
+//  Created by Scott Hoyt on 12/28/15.
+//  Copyright © 2015 Realm. All rights reserved.
+//
+
+import XCTest
+import Yaml
+@testable import SwiftLintFramework
+
+class YamlSwiftLintTests: XCTestCase {
```

Comment:
Yaml is sufficiently tested in https://github.com/behrang/YamlSwift IMO, so these tests (and therefore `test.yml` too) are unnecessary.